ddtrace/contrib/pytest/_utils.py

```diff
@@ -104,3 +123,24 @@ def _is_pytest_8_or_later() -> bool:
 
 def _pytest_version_supports_itr() -> bool:
     return _get_pytest_version_tuple() >= ITR_MIN_SUPPORTED_VERSION
+
+
+def _pytest_marked_to_skip(item: pytest.Item) -> bool:
+    """Checks whether Pytest will skip an item"""
+    if item.get_closest_marker("skip") is not None:
+        return True
+
+    return any([True for marker in item.iter_markers(name="skipif") if marker.args[0] is True])
+
+
+def _is_test_unskippable(item: pytest.Item) -> bool:
+    """Returns True if a test has a skipif marker with value false and reason ITR_UNSKIPPABLE_REASON"""
+    return any(
+        [
+            True
+            for marker in item.iter_markers(name="skipif")
+            if marker.args[0] is False
+            and "reason" in marker.kwargs
+            and marker.kwargs["reason"] is ITR_UNSKIPPABLE_REASON
+        ]
```

Comment:
@romainkomorndatadog ...I was going to propose a similar rewrite here, but I guess it's easier to read in this order than with the condition before the `for`. So feel free to ignore the previous one and close the conversation. :stuck_out_tongue:

Reply:
I think this works as well:
```suggestion
    return any(
        (marker.args[0] is False and marker.kwargs.get("reason") is ITR_UNSKIPPABLE_REASON)
        for marker in item.iter_markers(name="skipif")
    )
```
Will also change in next PR.